types/lnrpc.d.ts

```diff
@@ -489,6 +489,35 @@ export interface ListPaymentsResponse {
   payments: Payment[];
 }
 
+export interface NodeUpdate {
+  addresses: string[];
+  identityKey: string;
+  globalFeatures: string;
```

Comment:
Going off the docs, this should be a `Buffer`. Is it being returned as a string?

Reply:
yeah, it's returning a string